components/automate-chef-io/data/docs/api-static/03-tags.swagger.json

```diff
@@ -9,8 +9,8 @@
       "tags": ["ConfigMgmt"]
     },
     {
-      "name": "Nodes",
-      "tags": ["NodesService"]
+      "name": "Node Management Service",
```

Comment:
The other tags don't include "Service" in their names, should this be just `Node Management`?